routing/transit_world_graph.hpp

```diff
@@ -51,6 +55,13 @@ class TransitWorldGraph final : public WorldGraph
   std::unique_ptr<TransitInfo> GetTransitInfo(Segment const & segment) override;
 
 private:
+  static double MaxPedestrianTime(double distanceM)
+  {
+    // TODO: test and adjust constants.
```

Comment:
`// @todo(tatiana-kondakova) The constants should be adjust after testing.`

Reply:
done